From afade2b9e5f8a0fa696e9c8297b26c13b32daf85 Mon Sep 17 00:00:00 2001
From: Mousumi Jana <mousumix.jana@intel.com>
Date: Fri, 12 Aug 2016 19:19:35 +0530
Subject: [PATCH 4702/4706] ASoC: Intel: Boards: Fix checkpatch warning in IVI
 machine drivers

commit 14196847865827aa4246aeda02b1059121e9797e from
git://git.yoctoproject.org/linux-yocto-4.1

Fix checkpatch warnings in IVI machine drivers by adding one
blank line.

Change-Id: I8f1b3cfbfa66972357745326158651e2ba8a6251
Signed-off-by: Mousumi Jana <mousumix.jana@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/10895
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Tested-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
---
 sound/soc/intel/boards/bxtp_ivi_hu_rt298.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c b/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
index 8601dc2..eeba2fa 100644
--- a/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
+++ b/sound/soc/intel/boards/bxtp_ivi_hu_rt298.c
@@ -237,6 +237,7 @@ static int broxton_audio_probe(struct platform_device *pdev)
 {
 	int ret_val;
 	struct bxtp_ivi_hu_prv *drv;
+
 	broxton_rt298.dev = &pdev->dev;
 
 	drv = devm_kzalloc(&pdev->dev, sizeof(*drv), GFP_KERNEL);
-- 
1.7.5.4

